### ISSUE TITLE
Module not found: Error: Can't resolve 'storybook-readme'

### ISSUE THREAD

**Issue by enrique-ramirez** [tuchk4/storybook-readme#38]
I'm unable to use this addon at all. It doesn't seem to find it.

I've installed different addons (some namespaced `@storybook/module` and some not) and they all work, it's just storybook-readme the one I can't load. I'm not sure if it's a compatibility issue. I've tried with both the beta branch and the stable release to no avail.

I have quite a few dependencies in my package.json:

```json
  "devDependencies": {
    "@storybook/addon-actions": "^3.2.12",
    "@storybook/react": "^3.2.12",
    "babel-cli": "^6.26.0",
    "babel-core": "^6.26.0",
    "babel-eslint": "^8.0.1",
    "babel-loader": "^7.1.2",
    "babel-preset-env": "^1.6.1",
    "babel-preset-react": "^6.24.1",
    "css-loader": "^0.28.7",
    "eslint": "^4.9.0",
    "eslint-config-airbnb-base": "^12.1.0",
    "eslint-loader": "^1.9.0",
    "eslint-plugin-import": "^2.7.0",
    "postcss": "^6.0.13",
    "postcss-color-function": "^4.0.0",
    "postcss-cssnext": "^3.0.2",
    "postcss-custom-properties": "^6.2.0",
    "postcss-import": "^11.0.0",
    "postcss-loader": "^2.0.8",
    "prop-types": "^15.6.0",
    "raw-loader": "^0.5.1",
    "rimraf": "^2.6.2",
    "storybook-host": "^4.1.4",
    "storybook-readme": "3.1.0-beta2",
    "style-loader": "^0.19.0",
    "stylelint": "^8.2.0",
    "webpack": "^3.8.1",
    "yargs": "^9.0.1"
  },
  "dependencies": {
    "react": "^16.0.0",
    "react-dom": "^16.0.0"
  }
```

I've added the `import '@storybook/addon-actions/register'` line to my `.storybook/addons.js` file, but the moment I try to `import { withReadme, withDocs } from 'storybook-readme'` on my stories files, the error is thrown.

I'm not sure what's going on. Any help would be appreciated.

**Comment by jonattfin** [comment]
I confirm that is not working... I need it. Can somebody look over it, pleasee.

**Comment by ndelangen** [comment]
@tuchk4 I think you need to release, the last release on npm is quite old?
<img width="242" alt="screen shot 2017-10-28 at 01 41 05" src="https://user-images.githubusercontent.com/3070389/32128786-282d6588-bb81-11e7-9a73-fd9ccfcbf956.png">

**Comment by tuchk4** [comment]
duplicated issue #36

Try:
```js
import withReadme from 'storybook-readme/with-readme';
```

Also note that all new features (withDocs, footers) are still in beta at `3.1.0-beta2`

> I am waiting for #34 and will release version (not beta)

**Comment by tuchk4** [comment]
released `3.1.1`

Issue with import should be resolved. 

Reopen if any problems.
